bigquery_etl/shredder/delete.py

```diff
@@ -361,7 +408,7 @@ def create_job(client) -> bigquery.QueryJob:
                 """
             )
         run_tense = "Would run" if dry_run else "Running"
-        logging.debug(f"{run_tense} query: {query}")
+        logging.info(f"{run_tense} query: {query}")
```

Comment:
```suggestion
        logging.debug(f"{run_tense} query: {query}")
```
Revert unintentional change